source/generic-and-non-generic.rst

```diff
@@ -6,15 +6,14 @@
 Generic vs non-generic SYCL
 ***************************
 
-The SYCL programming API is split into two categories; generic SYCL
-and non-generic SYCL. Almost everything in the SYCL programming API
-is considered generic SYCL. However any usage of the
-``enum class backend`` is considered non-generic SYCL and should only
-be used for SYCL backend specialized code paths, as the identifiers
-defined in ``backend`` are implementation-defined.
+The SYCL programming API is split into two categories; generic SYCL and
+non-generic SYCL. Almost everything in the SYCL programming API is considered
+generic SYCL. However any usage of the ``enum class backend`` is considered
+non-generic SYCL and should only be used for SYCL backend specialized code paths
+, as the identifiers defined in ``backend`` are implementation-defined.
```

Comment:
I think you cannot add a line break between the comma and the previous word.  This causes the HTML render to insert a space between "paths" and the comma.

Reply:
Okay thanks, should be fixed in https://github.com/KhronosGroup/SYCL_Reference/pull/190/commits/28f738461df14f140f3c9c16f89fa0e98d4136d4.